src/lib/libwasmfs_fetch.js

```diff
@@ -94,7 +94,7 @@ addToLibrary({
       if (!response.ok) {
         throw response;
       }
-      var bytes = await response.bytes();
+      var bytes = await response['bytes']();
```

Comment:
I'm OK landing this this way.  I will see if I can find a way to teach closure about this as a followup.